The real problem is the `assert` you have changed:
```
2019-07-25 12:56:16,048 5824 ERROR openerp_test odoo.addons.account_invoice_factur-x.tests.test_facturx_invoice: ` AssertionError: None != 'factur-x.xml'
```
https://travis-ci.org/OCA/edi/jobs/563533926#L1198

--- a/account_invoice_factur-x/tests/__init__.py
+++ b/account_invoice_factur-x/tests/__init__.py
@@ -1,3 +1,3 @@
 # -*- coding: utf-8 -*-
 
-from . import test_facturx_invoice
+# from . import test_facturx_invoice

